optimade/validator/validator.py

```diff
@@ -398,18 +142,18 @@ def _setup_log(self):
         )
         self._log.addHandler(stdout_handler)
         if self.verbosity == 0:
-            self._log.setLevel(logging.CRITICAL)
+            self._log.setLevel(logging.WARNING)
```

Comment:
I was considering this, but I don't think we want to ever allow warnings to be ignored, the way we use them at the moment (to shout about tests that we should be running but aren't because we don't have them implemented ourselves, e.g. timestamp queries).

Reply:
Okay. Is there a way of minimizing the output for test purposes or other anyway? I.e., a `-1` option or something :sweat_smile:

Reply:
Hmmm, just ran it with verbosity 0 for the first time and I now get your point. I'll add that middle-ground verbosity.